src/components/api/api.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+export const instance = axios.create({
+  withCredentials: false,
+  baseURL: 'https://api.realworld.io/api/',
```

Comment:
Such things like 'baseURL' always depend on the environment (on a real project there are at least two environments: production and development/staging). That's why it'd be better to keep this var in the .env file (and the file itself should be in .gitignore) https://create-react-app.dev/docs/adding-custom-environment-variables/#:~:text=Adding%20Development%20Environment-,Variables,-In%20.env%E2%80%8B